### PR TITLE
Contact-me form

### PR DESCRIPTION
**Pull request by Marlyn-Mayienga**
# # Contact me form
In this module we have:
- created the contact form
- added the required fields such as placeholder for the input fields
- changed the color on the social icons on the footer
- changed the footer items alignment
- corrected the width and height of the input fields

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
...
+          action="https://formspree.io/f/xvolvllr"
+          method="POST"
+        >
           <input
             type="text"
             id="fullname"
             name="fullname"
             placeholder="Enter Full Name"
+            maxlength="30"
           />
```
Comment (thecodechaser) [inline index.html]: - [ ] A user cannot submit the form if one or more of the fields is empty, Please also add `required` attributeto the name field so the user can not submit the form if the name field is empty.

File: index.html [hunk 2]
```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
       </section>
       <section class="contact">
         <p class="contact-title desktop">Let's chat!</p>
-        <p class="contact-title mobile">CONTACT US</p>
+        <p class="contact-title mobile">CONTACT ME</p>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please change the `<p>` tag with `<hx>` like h2, h3, h4. Lets follow the best code practices

File: index.html [hunk 3]
```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
...
         <p class="contact-description">
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
           with it
         </p>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please use the correct font-weight for the heading and align the paragraph text in the center and make sure it matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design.

### Your design 

image

### Figma design

image

File: index.html [hunk 4]
```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
...
+          class="contact-form"
+          action="https://formspree.io/f/xvolvllr"
+          method="POST"
+        >
           <input
             type="text"
             id="fullname"
             name="fullname"
             placeholder="Enter Full Name"
+            maxlength="30"
```
Comment (thecodechaser) [inline index.html:249]: - [ ] Please use correct paddings for the input fields and make sure it matches with the Figma design.

### Your design

image

### Figma design

image

File: index.html [hunk 5]
```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
...
+
           <button type="submit" class="send-button">Send</button>
         </form>
       </section>
-
+      <div class="divider-full"></div>
       <footer class="second-footer">
         <ul>
           <li>
             <a href="https://github.com/Marlyn_Mayienga"
```
Comment (thecodechaser) [inline index.html:296]: - [ ] Please increase the size for the social media icons and make sure it matches with the Figma design.

### Your design

image

### Figma design

image

File: index.html [hunk 6]
```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
...
-          Write your message here
-      </textarea
+          <!-- <textarea
+            placeholder="Write your message here..."
+            rows="5"
+            cols="50"
+            name="description"
+            maxlength="500"
+            required
           >
```
Comment (thecodechaser) [inline index.html]: - [ ] [optional] I would suggest removing these comments from the code to make your code clean.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Marlyn-Mayienga 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
- [ ] Please make sure that every member from your coding partner group has contributed to this project, I can only see your commits, I am requesting some changes Implement the changes with your group in pair programming we will check the commit history.
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @Marlyn-Mayienga, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**